Makefile

```diff
@@ -103,7 +103,6 @@ cluster-api: $(CONTROLLER_GEN)
 	rm -rf cmd/install/assets/cluster-api/*.yaml
 	$(CONTROLLER_GEN) $(CRD_OPTIONS) paths="./vendor/sigs.k8s.io/cluster-api/api/..." output:crd:artifacts:config=cmd/install/assets/cluster-api
 	$(CONTROLLER_GEN) $(CRD_OPTIONS) paths="./vendor/sigs.k8s.io/cluster-api/exp/api/..." output:crd:artifacts:config=cmd/install/assets/cluster-api
-	$(CONTROLLER_GEN) $(CRD_OPTIONS) paths="./vendor/sigs.k8s.io/cluster-api/exp/addons/api/..." output:crd:artifacts:config=cmd/install/assets/cluster-api
```

Comment:
How's this change related to this PR?
can we also get rid of cluster-api-provider-aws/v2/exp/api/?

Reply:
Because it does not exists.
The `exp/api/...` exists and If I get rid of that some CRDs will not be generated:

```
deleted:    cmd/install/assets/cluster-api-provider-aws/infrastructure.cluster.x-k8s.io_awsfargateprofiles.yaml
deleted:    cmd/install/assets/cluster-api-provider-aws/infrastructure.cluster.x-k8s.io_awsmachinepools.yaml
deleted:    cmd/install/assets/cluster-api-provider-aws/infrastructure.cluster.x-k8s.io_awsmanagedmachinepools.yaml
```

I believe if those CRDs are not there could cause a crash with older versions of the HC, isn't it?

Reply:
Older capi versions might also require the ones that you are deleting above, can we validate that?

Reply:
Getting back the line to generate the CRDs for `clusterresourcesetbindings` and `clusterresourcesets`.